enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/configuration/IdentityVerificationConfig.java

```diff
@@ -51,16 +53,22 @@ public class IdentityVerificationConfig {
     @Value("${enrollment-server-onboarding.presence-check.cleanupEnabled:false}")
     private boolean presenceCheckCleanupEnabled;
 
-    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:1}")
-    private int dataRetentionTime;
+    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:1h}")
```

Comment:
Since it is already type of `Duration`, I guess that _hours_ may be omit from property key.

Reply:
Right, I was about to change this, but forgot about this place.